docs/appendix.md

```diff
@@ -0,0 +1,275 @@
+
+#### Generating the native code artifacts
+
+Once you have a schema file for your native modules or components, you can use a second script to generate the actual native code artifacts for your library. You can use the same schema file generated by the previous script.
+
+```bash
+node node_modules/react-native/scripts/generate-specs-cli.js \
+  --platform <ios|android> \
+  --schemaPath <generated_schema_json_file> \
+  --outputDir <output_dir> \
+  [--libraryName library_name] \
+  [--javaPackageName java_package_name] \
+  [--libraryType all(default)|modules|components]
+```
+
+> [!Note]
+> The output artifacts of the Codegen are inside the build folder and should not be committed.
+> They should be considered only for reference.
+
+##### Example
+
+The following is a basic example of invoking the Codegen script to generate native iOS interface code for a library that provides native modules. The JavaScript spec sources for this library are located in a `js/` subdirectory, and this library’s native code expects the native interfaces to be available in the `ios` subdirectory.
+
+```bash
+# Generate schema - only needs to be done whenever JS specs change
+node node_modules/react-native-codegen/lib/cli/combine/combine-js-to-schema-cli.js /tmp/schema.json ./js
+
+# Generate native code artifacts
+node node_modules/react-native/scripts/generate-specs-cli.js \
+  --platform ios \
+  --schemaPath /tmp/schema.json \
+  --outputDir ./ios \
+  --libraryName MyLibSpecs \
+  --libraryType modules
+```
+
+In the above example, the code-gen script will generate several files: `MyLibSpecs.h` and `MyLibSpecs-generated.mm`, as well as a handful of `.h` and `.cpp` files, all located in the `ios` directory.
+
+## V. Note on Existing Apps
+
+This guide provides instructions for migrating an application that is based on the default app template that is provided by React Native. If your app has deviated from the template, or you are working with an application that was never based off the template, then the following sections might help.
+
+### Finding your bridge delegate
+
+This guide assumes that the `AppDelegate` is configured as the bridge delegate. If you are not sure which is your bridge delegate, then place a breakpoint in `RCTBridge` and `RCTCxxBridge`, run your app, and inspect `self.delegate`.
+
+---
+
+> [!IMPORTANT]
+> This documentation is still experimental and details are subject to changes as we iterate.
+> Feel free to share your feedback on this [discussion](https://github.com/reactwg/react-native-new-architecture/discussions/8).
+>
+> Moreover, it contains **several manual steps**. Please note that this won't be representative of the final developer experience once the New Architecture is stable. We're working on tools, templates and libraries to help you get started fast on the New Architecture, without having to go through the whole setup.
```

Comment:
Also these three lines can be removed: we dramatically reduced the number of manual steps to migrate to the new architecture.

Reply:
I've kept this as a general disclaimer everywhere -- I think we should remove it only when everything is "final"